pylint_django/tests/input/func_noerror_string_foreignkey.py

```diff
@@ -0,0 +1,10 @@
+"""
+Checks that PyLint correctly handles string foreign keys
+https://github.com/PyCQA/pylint-django/issues/243
+"""
+#  pylint: disable=missing-docstring
+from django.db import models
+
+
+class Book(models.Model):
+    author = models.ForeignKey("test_app.Author", models.CASCADE)
```

Comment:
Let's also add another FK reference to "auth.User" - because django apps don't follow the usual convention of app name vs. module name. That's was one of the reasons we didn't deal with string FKs for so long.